clients/banking/src/components/AccountMembersDetailsCardList.tsx

```diff
@@ -90,14 +84,14 @@ export const AccountMembersDetailsCardList = ({
           )
         : undefined,
     } as const;
-  }, [params.cardSearch, params.cardStatuses, params.cardType]);
+  }, [params.cardSearch, params.cardStatus, params.cardType]);
 
   const hasFilters = Object.values(filters).some(isNotNullish);
 
-  const statuses = match(filters.statuses)
-    .with(["Active"], () => ACTIVE_STATUSES)
-    .with(["Canceled"], () => CANCELED_STATUSES)
-    .otherwise(() => [...ACTIVE_STATUSES, ...CANCELED_STATUSES]);
+  const statuses = match(filters.status)
+    .with("active", () => ACTIVE_STATUSES)
+    .with("canceled", () => CANCELED_STATUSES)
+    .otherwise(() => ACTIVE_STATUSES);
```

Comment:
why the capitalization change here?

Reply:
to be consistent with the other costs we use elsewhere but I don't have a strong opinion on this point, up to you